src/style.css

```diff
@@ -0,0 +1,97 @@
+* {
+  padding: 0;
```

Comment:
Only `box-sizing` allow for `*` selector. Please move `padding` property for `body` tag
```suggestion
```